### ISSUE TITLE
File names should not be percent-encoded when uploading file

### ISSUE THREAD

**Issue by mitchellarchibald** [a85/POSTMan-Chrome-Extension#1768]
If I make a POST request, selecting form-data as the body type and uploading a file called `with spaces.csv`, Postman percent-encodes the file name:

```
Content-Disposition: form-data; name="file"; filename="with%20spaces.csv"
```

This is incorrect; header fields in HTTP messages and MIME parts are not to be percent-encoded. (except when using the extended parameter notation defined in RFC 5987)  The correct header should be:

```
Content-Disposition: form-data; name="file"; filename="with spaces.csv"
```

The best resource is probably [http://greenbytes.de/tech/tc2231/](http://greenbytes.de/tech/tc2231/), which collates all the RFCs relevant to the Content-Disposition header and includes an extensive set of examples.

**Comment by abhijitkane** [comment]
@mitchellarchibald Thanks for reporting this. We'll fix this with the upcoming release of the app.

**Comment by abhijitkane** [comment]
@mitchellarchibald This has been fixed in v4.4.2 of the app (Chrome only). Can you update (https://www.getpostman.com/docs/updating) and check?

**Comment by mitchellarchibald** [comment]
@abhijitkane I've tested with the latest version and can confirm the filename is now correct.  Thanks for your help!
